.env.example

```
@@ -5,4 +5,10 @@ DB_DATABASE=howdusty
 DB_USERNAME=root
 DB_PASSWORD=
 
-GITHUB_ACCESS_TOKEN=
\ No newline at end of file
+GITHUB_ACCESS_TOKEN=
+
+ONLYDUST_HOST=
+ONLYDUST_PORT=
+ONLYDUST_DATABASE=
+ONLYDUST_USERNAME=
+ONLYDUST_PASSWORD=
```

```suggestion
ONLYDUST_DB_HOST=
ONLYDUST_DB_PORT=
ONLYDUST_DB_DATABASE=
ONLYDUST_DB_USERNAME=
ONLYDUST_DB_PASSWORD=
```

it's unclear what these var relate to, specifying the "_DB_" will help people figure out